CONTRIBUTING.md

```diff
@@ -165,3 +165,27 @@ general guidelines for the community as a whole:
   respond to an email you are potentially sending to a large number of
   people.  Please consider this before you update.  Also remember that
   nobody likes spam.
+
+## FAQ
+
+**Is it fine to go with kotlin for my contribution?**
+
+**Is there an architectural design for the presentation layer?**
+
+**Is there an architectural design for the data layer?**
```

Comment:
Design as in design documentation? No, there isn't.
Design as in 'meaningful structuring of code' We did our best to achieve that ;-)

Reply:
Design as 'meaningful structuring of code' + use of designpatterns like mvvm/mvp/mvc for the ui layer or repository pattern, use cases for the data or domain layer.

Reply:
We have proper separation between data model (see model package) and presentation (see ui package). We don't separate matters further, so no mvvm/mvp/mvc. As the application isn't overly complex I think this is fine.